turbopack/crates/turbo-tasks-fs/src/rope.rs

```diff
@@ -536,11 +539,30 @@ impl InnerRope {
             }
         }
     }
+
+    fn into_bytes(mut self, len: usize) -> Result<Cow<'static, [u8]>> {
+        if self.0.is_empty() {
+            return Ok(Cow::Borrowed(EMPTY_BUF));
+        } else if self.0.len() == 1 {
+            let data = Arc::try_unwrap(self.0);
+            match data {
+                Ok(data) => return data.into_iter().next().unwrap().into_bytes(len),
+                Err(data) => {
+                    self.0 = data;
+                }
+            }
+        }
+
+        let mut read = RopeReader::new(&self, 0);
+        let mut buf = Vec::with_capacity(len);
+        read.read_to_end(&mut buf)?;
```

Comment:
afaict RopeReader never returns an Err but this api does due to the `Read` trait.

shouldn't we just `.unwrap` or `.expect` instead of `?` to avoid complicating our call signature with impossible conditions?